src/reducers/profile-view.js

```diff
@@ -1014,7 +1014,7 @@ export const selectorsForThread = (
       }
     );
     const getIsNetworkChartEmptyInFullRange = createSelector(
-      getTracingMarkers,
+      getSearchFilteredTracingMarkers,
```

Comment:
The `getIsNetworkChartEmptyInFullRange` component got the unfiltered markers and did not show any markers, just the empty container.